crates/aptos/src/move_tool/bytecode.rs

```diff
@@ -80,6 +89,11 @@ pub struct BytecodeCommand {
 
     #[clap(flatten)]
     pub(crate) prompt_options: PromptOptions,
+
+    /// When `--bytecode-path` is set with this option,
```

Comment:
Could you post what the output looks like (either here or in the PR description)?

Reply:
done

Reply:
From the output, it looks like the first level is keys mapped to just strings, instead of JSON all the way through (in the second level, you can see `\"` escaping, making them plain strings instead of JSON). This is also why they are not prettified all the way through (and instead, there are `\n`s in the string).

Reply:
I changed how it is printed out, which is shown in the PR description.